code/game/objects/items/implants/implant_bb.dm

```diff
@@ -0,0 +1,65 @@
+/obj/item/implant/bloodbrother
+	name = "communication implant"
+	desc = "Use this to communicate with your fellow blood brother(s)."
+	icon = 'icons/obj/radio.dmi'
+	icon_state = "headset"
+	item_color = "r"
+	var/list/linked_implants // All other implants that this communicates to
+
+/obj/item/implant/bloodbrother/Initialize()
+	. = ..()
+	linked_implants = list()
+
+/obj/item/implant/bloodbrother/activate()
+	. = ..()
+	if(linked_implants.len)
+		var/input = stripped_input(imp_in, "Enter a message to communicate to your blood brother(s).", "Radio Implant", "")
+		if(!input)
+			return
+
+		var/preliminary_message = "<span class='bold'>[input]</span>" //apply basic color/bolding
```

Comment:
Don't the actual messages themselves on radio usually have the message in non bold and only the name in bold?